functionsSrc/addToNewsletterList/addToNewsletterList.js

```diff
@@ -0,0 +1,57 @@
+const request = require('request');
+
+module.exports.handler = (event, context, callback) => {
+  const { email } = event.queryStringParameters;
+  let errorMessage = null;
+
+  if (!email) {
+    errorMessage = 'No EMAIL supplied';
+    console.log(errorMessage);
+    callback(errorMessage);
+  }
+
+  const data = {
+    email_address: email,
+    status: 'subscribed',
+    merge_fields: {},
+  };
+
+  const subscriber = JSON.stringify(data);
+  console.log('Sending data to mailchimp', subscriber);
```

Comment:
Console.log can be removed?

Reply:
Netlify functions are a complete black box, apart from the console via the Netlify CMS (hence the huge number of commits!). Would prefer to leave them in.